src/c_wrapper.jl

```diff
@@ -47,8 +47,22 @@ function SCS_solve(T::Union{Type{Direct}, Type{Indirect}},
 
     managed_matrix = ManagedSCSMatrix(m, n, A)
     matrix = Ref(SCSMatrix(managed_matrix))
-    settings = Ref(SCSSettings(;options...))
+    settings = Ref(SCSSettings())
     data = Ref(SCSData(m, n, Base.unsafe_convert(Ptr{SCSMatrix}, matrix), pointer(b), pointer(c), Base.unsafe_convert(Ptr{SCSSettings},settings)))
+    SCS_set_default_settings!(T, data)
+
+    opts = Dict(options)
+
+    all_kwargs = [:scale, :rho_x, :max_iters, :eps, :alpha, :cg_rate, :acceleration_lookback, :normalize, :verbose, :warm_start]
+    for k in keys(opts)
+        k in all_kwargs || throw(ArgumentError("Unknown keyword passed to SCS solver: $k"))
+    end
+
+    for param in keys(opts)
+        # when dropping support for 0.6 could be replaced with setproperty!
```

Comment:
What could be replaced exactly? To make sure this happens, you could use a VERSION if/else.